Automattic/wp-calypso
# Inline Help: Show/hide popover with redux actions

#### Changes proposed in this Pull Request

* Sometimes the visibility of the inline help popover needs to be controlled by something outside of the component. This helps you show/hide the popover with corresponding redux actions. You can also check if the popover is currently displayed with a new redux selector.

#### Testing instructions



* Run the unit test against the new added actions, reducers and selectors.
```
npm run test-client client/state/inline-help
```
* To check it visually, go to a Calypso page that shows the Inline help component(i.e `/view/YOUR_SITE`).
* Check how the `isPopoverVisible` prop of the Inline Help changes as you toggle the popover. When it shows up, the prop should be `true`. Otherwise, `false`.

Refs. https://trello.com/c/taGAfTIJ/303-pop-up-inline-help-on-site-preview
